### DISCUSSION TITLE
DynUNet configuration to reproduce BraTS2020 nnUnet baseline's architecture

### DISCUSSION THREAD

**Discussion by LouisDo2108** [Project-MONAI/MONAI#6302] [Q&A]
Hi! I am using MONAI to reimplement the nnU-Net baseline's architecture from this paper: https://arxiv.org/pdf/2011.00848.pdf
Here is the overview of the architecture.
![image](https://user-images.githubusercontent.com/54947727/230130517-3833363c-af53-4fc2-9cee-d63552e8989b.png)
My current configuration is
```
kernel_size = [3]*6
strides = [1] + [2]*5
net = DynUNet(
    spatial_dims=3,
    in_channels=4,
    out_channels=4,
    kernel_size=kernel_size,
    strides=strides,
    upsample_kernel_size=[1]*5,
    norm_name="instance",
    deep_supervision=True,
    deep_supr_num=3,
)
```
Is this correct? Thank you!

**Comment by KumoLiu** [comment]
Hi @yiheng-wang-nv, could you please share some comments here? Thanks in advance!
